packages/procps/sysinfo.c.patch

```diff
@@ -0,0 +1,40 @@
+--- procps-3.3.17.orig/proc/sysinfo.c	2021-02-09 17:11:25.000000000 +0700
```

Comment:
Why copying https://github.com/vhqtvn/termux-packages/blob/for-merge-20210528/packages/procps/proc-sysinfo.c.patch?

Reply:
This seems to be problem of merging my edit and master, i'll remove this package.